src/Componenets/App/App.css

```diff
@@ -0,0 +1,45 @@
+.container{
+	/*border: 1px solid red;*/
+}
+
+.card-header{
+	box-sizing: content-box;
+	margin: 0;
+	padding: 0;
+}
+
+
+
+#signup,#signin{
+	color: black;
+	font-size: 2em;
+	text-decoration: none;
+}
+
```

Comment:
keep only one line gap between `CSS` classes rules and properties. It would keep code clean and less space occupying.